lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -754,3 +758,28 @@ def cohort_students_and_upload(_xmodule_instance_args, _entry_id, course_id, tas
     upload_csv_to_report_store(output_rows, 'cohort_results', course_id, start_date)
 
     return task_progress.update_task_state(extra_meta=current_step)
+
+
+def push_student_responses_to_s3(_xmodule_instance_args, _entry_id, course_id, _task_input, _action_name):
```

Comment:
@kluo what's up with the `action_name` parameter here? It's unused - will it cause problems to call it `_action_name` here?